kvrocks.conf

```diff
@@ -597,6 +597,30 @@ rocksdb.enable_blob_garbage_collection yes
 # Default: 25
 rocksdb.blob_garbage_collection_age_cutoff 25
 
+
+# The purpose of following three options are to dynamically adjust the upper limit of
+# the data that each layer can store according to the size of the different 
+# layers of the LSM. Enabling this option will bring some improvements in
+# deletion efficiency and space amplification, but it will lose a certain 
+# amount of read performance.
+# If you want know more details about Levels' Target Size, you can read RocksDB wiki: 
+# https://github.com/facebook/rocksdb/wiki/Leveled-Compaction#levels-target-size
+#
+# Default: no
+rocksdb.level_compaction_dynamic_level_bytes no
+
+# The total file size of level-1 sst.
+#
+# Default: 256 M
+rocksdb.max_bytes_for_level_base 256
```

Comment:
since this config has `bytes` word, we should use byte unit, right? or we rename it to level_base_size? maybe rocksdb name is better?

This commit is not released in newest version, we have chances to change.

Reply:
Yeah, I also think `bytes` is better.